### ISSUE TITLE
Dialog children is not rendering, and TypeScript gives an error for any child.

### ISSUE THREAD

**Issue by omer-to** [douglasjunior/react-native-simple-dialogs#57]
- What version of React Native?
0.63.2
- What version of the library?
1.3.0
- iOS/Android version?
iPhone 11, iOS 13.7
- Did the problem happen after updating React Native?
Nope.
- Are you using the library for the first time?
Yes.
- What code snippet did you use?
```
<Dialog
	visible={ true }
	title="Password"
	onTouchOutside={ () => this.setState({ dialogVisible: false }) }
        animationType='fade'
>
	<View style={ { flex: 0.2 } }>
		<View style={ styles.dialogButtons } >
			<Button title='Cancel' onPress={ () => { } } />
			<Button title='Copy' onPress={ () => { } } />
		</View>
	</View>

</Dialog>
```
As soon as I insert any element as a child of Dialog component, TypeScript starts complaining. More than that, I am not seeing anything on the dialog except for the title.
More interestingly, though in an unexpected behavior, components do appear on the screen when I wrap them inside the <Text> component. However, I do see that in all of your examples in README, you put a View element as the first child of the Dialog component.

Here is the TypeScript error:

> No overload matches this call. 
> Overload 1 of 2, '(props: Readonly<DialogProps>): Dialog', gave the following error.
> Type 'Element' is not assignable to type 'ComponentClass<any, any> | FunctionComponent<any> | (ComponentClass<any, any> & string) | (ComponentClass<any, any> & number) | ... 11 more ... | (FunctionComponent<...> & ReactPortal)'.
> Type 'Element' is not assignable to type 'FunctionComponent<any> & ReactPortal'.
> Type 'Element' is not assignable to type 'FunctionComponent<any>'.
> Overload 2 of 2, '(props: DialogProps, context?: any): Dialog', gave the following error.
> Type 'Element' is not assignable to type 'ComponentClass<any, any> | FunctionComponent<any> | (ComponentClass<any, any> & string) | (ComponentClass<any, any> & number) | ... 11 more ... | (FunctionComponent<...> & ReactPortal)'.\n      Type 'Element' is not assignable to type 'FunctionComponent<any> & ReactPortal'.
> Type 'Element' is not assignable to type 'FunctionComponent<any>'.",
> 
> 
> The expected type comes from property 'children' which is declared here on type 'IntrinsicAttributes & IntrinsicClassAttributes<Dialog> & Readonly<DialogProps> & Readonly<{ children?: ReactNode; }>'",
> 			"resource": "node_modules/react-native-simple-dialogs/index.d.ts"

**Comment by AbnerBaluyut** [comment]
I got the same issue ^

**Comment by douglasjunior** [comment]
About TypeScript I can't help right now, our projects are all JavaScript. If anyone can create an sample project with TypeScript I can take a look.

About the `children` that is not rendering, does the problem also happen in the [sample project](https://github.com/douglasjunior/react-native-simple-dialogs/tree/master/Sample)?

**Comment by jussirantala** [comment]
> About TypeScript I can't help right now, our projects are all JavaScript. If anyone can create an sample project with TypeScript I can take a look.
> 
> About the `children` that is not rendering, does the problem also happen in the [sample project](https://github.com/douglasjunior/react-native-simple-dialogs/tree/master/Sample)?

There is index.d.ts file in this project ...

**Comment by douglasjunior** [comment]
Fixed on #58
